glide.yaml

```diff
@@ -0,0 +1,5 @@
+package: github.com/coinculture/dev-tools
+import:
+- package: github.com/codegangsta/cli
+- package: github.com/pkg/errors
+- package: github.com/spf13/cobra
```

Comment:
why are we using two different cli packages? (codegangsta & cobra). I really don't think we need `pkg/errors`; that's really for advanced projects IMO. I also think we should remain "pure" by not importing anything unless absolutely necessary (not yet the case). Using a cli package makes sense for large projects, but this repo is unlikely to go in that direction ... at most, every tool will have a couple commands and probably zero sub-commands, so using a cli package is somewhat of an abuse since the standard lib can handle the (non-existent) complexity. Any tool in this repo that does become large enough to require importing external packages would likely eventually be split into its own repo but that's really another conversation. Refer to https://github.com/dominikh/go-tools which accomplishes a ton without using external packages